vars/maven.groovy

```diff
@@ -132,9 +135,22 @@ def uploadLocalArtifacts(String mvnUploadCredsId, String artifactDir, String rep
     }
 }
 
+def getLatestArtifactVersionFromRepository(String repositoryUrl, String groupId, String artifactId) {
+    def groupIdArtifactId = "${groupId.replaceAll("\\.", "/")}/${artifactId}"
+    def metadata = new XmlSlurper().parse("${repositoryUrl}/${groupIdArtifactId}/maven-metadata.xml")
```

Comment:
maybe could be extracted as it is repeated in both methods

Reply:
done, thanks!